.goreleaser.yml

```diff
@@ -127,7 +128,25 @@ dockers:
     build_flag_templates:
       - "--platform=linux/arm64"
       - "--build-arg=VAULT_VERSION={{.Env.VAULT115}}"
-
+  ##############
+  #  VAULT 1.16
+  ##############
+  - goos: linux
+    goarch: amd64
+    image_templates:
+      - "contentsquareplatform/vault-plugin-database-clickhouse:{{ .Env.VAULT116 }}-{{ .Tag }}-amd64"
+    use: buildx
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--build-arg=VAULT_VERSION={{.Env.VAULT115}}"
```

Comment:
It's normal that one time you have `VAULT116` and the other time `VAULT115`

Reply:
no fixed thanks